package com.jessica.javabase.day01;

public class VariableDemo2 {
    /*
	我方：叉子		对方：长手
	攻击：220		攻击：210
	防御：85		防御：80
	血量：1012.5	血量：1223.3
	技能加成： 1.2	技能加成：1.3

	技能造成伤害的公式：攻击力 * 技能加成 - 对方防御力
	普攻造成伤害的公式：攻击力 - 对方防御力

	计算：
		我方第一次进行普通攻击，造成多少伤害，对方还剩余多少血量？
		我方第二次进行技能攻击，造成多少伤害，对方还剩余多少血量？

	规则：经常发生改变的数据，用变量记录
*/
    public static void main(String[] args) {
        double myAttack = 220;
        double myDefense = 85;
        double myBlood = 1012.5;
        double mySkill = 1.2;
        double enemyAttack = 210;
        double enemyDefense = 80;
        double enemyBlood = 1223.3;
        double enemySkill = 1.3;
        double myAttackDamage = myAttack - enemyDefense;
        double enemyBloodAfterMyAttack = enemyBlood - myAttackDamage;
        System.out.println(enemyBloodAfterMyAttack);
        double mySkillDamage = myAttack * mySkill - enemyDefense;
        double enemyBloodAfterMySkill = enemyBlood - mySkillDamage;
        System.out.println(enemyBloodAfterMySkill);
    }


}
